src/components/atoms/UiRange/UiRange.vue

```diff
@@ -120,37 +120,28 @@ const props = defineProps({
     default: 1,
   },
   /**
-   * Use this props to pass attrs for decrement UiButton
+   * Use this props to pass attrs for UiNumberStepper
    */
-  buttonDecrementAttrs: {
```

Comment:
use `numberStepperAttrs` to pass data to `UiNumberStepper` and remove `buttonDecrementAttrs` and `buttonIcrementAttrs`